### ISSUE TITLE
Backtesting strategy ta4j extremely slow

### ISSUE THREAD

**Issue by Freeongoo** [cassandre-tech/cassandre-trading-bot#946]
I have a strategy that trades on 5 minute candles. To improve the test results, I download historical data for 1 minute in order to understand the price movement within 5 minutes. As a result, testing takes a very long time. When testing an interval of half a month - about an hour.

Has this problem been known? Is there any way to speed up testing?

**Comment by straumat** [comment]
@Freeongoo can you give me the time it takes and the number of tickers your import ?

**Comment by Freeongoo** [comment]
@straumat when testing interval: '20211108-20220124_BTC-USDT_min15.tsv' (7392 candles): 30 min

**Comment by straumat** [comment]
Ok, i never made such bit tests :( I will check but I don't see how I could go faster as each treatment must be done before sending a new ticker...

**Comment by straumat** [comment]
Removed ta4j support - Will work on speed improvement on another ticket
